lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -188,20 +207,39 @@ class UserProfileBloc {
     action.resolve(pinCode == action.enteredPin);
   }
 
+  Future _createBackupPhrase(CreateBackupPhrase action) async {
+    await _secureStorage.write(key: 'backupKey', value: bip39.mnemonicToEntropy(action.backupPhrase));
+    action.resolve(null);
+  }
+
+  Future _validateBackupPhrase(ValidateBackupPhrase action) async {
+    var backupPhrase = await _secureStorage.read(key: 'backupKey');
+    String enteredBackupPhrase;
+    try {
```

Comment:
We don't need try/catch here. This handler is invoked within an action and better to only throw exception here which will make the action resolve to error.
In the UI we can catch error to handle validation failed.

Reply:
This method is now obsolete and removed [here](https://github.com/breez/breezmobile/pull/173/commits/d8b131b0ffa327af188f8ce98a6581d2305d12ec)